ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -321,7 +322,12 @@ export default function PrivacySettings() {
             ])}
           />
         </div>
-        <Button type="primary" rounded onClick={handleSubmit}>
+        <Button
+          variant={BUTTON_VARIANT.PRIMARY}
+          style={{ width: '100%' }}
+          rounded
+          onClick={handleSubmit}
```

Comment:
Check out the Button docs to align the prop types some of these have props and some of these aren't accepted in the `Button` component https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-button--docs

```suggestion
          variant={BUTTON_VARIANT.PRIMARY}
          block
          onClick={handleSubmit}
```

Reply:
Got it.
It is nicely explained in the docs.

Thanks a ton for patiently suggesting so many changes.

Will be more careful from now on.
🙏🏼
Regards,
Harsh